server/game/chatcommands.js

```diff
@@ -329,6 +330,29 @@ class ChatCommands {
 
         this.game.queueStep(new RematchPrompt(this.game, player));
     }
+
+    firstPlayer(player, args) {
+        const firstPlayerName = args[1];
+        if (this.game.startingHandsDrawn) {
+            this.game.addAlert('danger', 'Cannot change first player on this stage of game');
+            return;
+        }
+
+        const players = this.game.getPlayers();
+
+        for (const p of players) {
+            if (p.name === firstPlayerName) {
+                this.game.activePlayer = p;
+                this.game.addAlert('info', '{0} changed first player to {1}', player, p);
+                return;
+            }
+        }
+        this.game.addAlert(
+            'danger',
+            'Cannot change first player: player {0} not exists',
```

Comment:
Small thing, but this should say "player {0} does not exist", I'll change it for you :)